go.mod

```diff
@@ -5,40 +5,47 @@ go 1.21
 require golang.org/x/exp v0.0.0-20230522175609-2e198f4a06a1 // indirect
 
 require (
-	github.com/bytedance/sonic v1.9.1 // indirect
-	github.com/chenzhuoyu/base64x v0.0.0-20221115062448-fe3a3abad311 // indirect
-	github.com/gabriel-vasile/mimetype v1.4.2 // indirect
+	github.com/bytedance/sonic v1.11.9 // indirect
```

Comment:
can you please move the imports into a dedicated go.mod, for example/

I would like to keep my packages as lightweight as possible

Reply:
yes, I change project layout, but I use go.work to split example and slog-gin package, please review again, If have any problems, I will try to change it

Reply:
Seems good to me!
Thanks for your kind help!